modules/infinite_blocks/src/Plugin/Block/InfiniteHeaderMediaBlock.php

```diff
@@ -20,9 +20,10 @@ class InfiniteHeaderMediaBlock extends BlockBase {
    * {@inheritdoc}
    */
   public function build() {
+    $entity = NULL;
     if ($node = \Drupal::request()->attributes->get('node')) {
       /* @var Node $node */
-      $entity = $node;
+      $entity = is_object($node) ? $node : Node::load($node);
```

Comment:
Usually yes, but as line 24 already tries to load the node directly via request attribute, it should be enough at this point. Furthermore this is an exceptional case coming from the routing for node revisions.